CONTRIBUTING.rst

```diff
@@ -126,7 +135,7 @@ Before you submit a pull request, check that it meets these guidelines:
    Travis-CI.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
-   feature to the list in README.rst.
+   feature to the list in README.md.
 3. The pull request should work for Python 3.6 and later. Check
    https://travis-ci.org/elfi-dev/elfi/pull_requests
```

Comment:
should this be updated too?

Reply:
Absolutely! That's actually the thing I was supposed to fix!